ansible/local_testing/README.md

```diff
@@ -37,12 +61,25 @@ vms:
 
 # Fixes/Notes
 
+
+### Ansible cannot decrypt the files encrypted with ansible vault.
+
+* The `ansible\roles\certbot\vars\main.yaml` and `ansible\roles\pydis-users\vars\main.yaml` files have been encrypted with ansible vault due to their sensitive content.
```

Comment:
Should we use forward slashes here? On both GitHub and any non-Windows system those are used, so you could also e.g. link to them by using forward slashes. I would find it better for consistency

Reply:
Sounds good to me, I'll make the change